docker/scripts/prow-entrypoint-main.sh

```diff
@@ -69,8 +67,4 @@ set +x
 EXIT_VALUE=$?
 set -x
 
-# We cleanup in the trap as well, but just in case try to clean up here as well
-# shellcheck disable=SC2046
-cleanup
```

Comment:
why remove the cleanup?

Reply:
nevermind, I see the comment now

Reply:
the commit message explains it :)
it's done anyway, by a TRAP